gridcomps/ExtData3G/tests/Test_ExtDataGridComp.pf

```diff
@@ -0,0 +1,57 @@
+module Test_ExtDataGridComp
```

Comment:
@bena-nasa I have created a new include file that lets us use `_RC` in pFUnit tests.  See `./include/MAPL_TestErr.h`

Reply:
ok, I'll take a look and cleanup in subsequent PR